model-server/src/main/kotlin/org/modelix/model/server/handlers/AboutInformationHandler.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2024.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.modelix.model.server.handlers
+
+import io.ktor.server.application.Application
+import io.ktor.server.application.call
+import io.ktor.server.resources.get
+import io.ktor.server.response.respond
+import io.ktor.server.routing.routing
+import org.modelix.api.public.AboutInformation
+import org.modelix.api.public.Paths
+import org.modelix.model.api.MODELIX_VERSION
+
+object AboutInformationHandler {
```

Comment:
What's the idea behind this being a singleton? The other handlers are regular classes.

Reply:
We do not need class properties, so we do not need a class. Therefore I keept it simple.